cfbs/commands.py

```diff
@@ -392,6 +405,21 @@ def _get_module_by_name(name) -> dict:
                 return module
         return None
 
+    def _build_remove_module_user_prompt(module):
```

Comment:
I expected this to _build_ the prompt (i.e. return the prompt string), but this function also does the prompting, I'd remove _build_: 

```suggestion
    def _remove_module_user_prompt(module):
```